mix.exs

```diff
@@ -41,6 +41,7 @@ defmodule GoogleSearchDataViewer.MixProject do
   defp deps do
     [
       {:bcrypt_elixir, "~> 3.0"},
+      {:calendar, "~> 1.0.0"},
```

Comment:
Why we need this? 🤔  Can you try this one https://hexdocs.pm/elixir/1.13/Calendar.html?